testing.json

```diff
@@ -0,0 +1,9 @@
//api.github.com/orgs/DSC-JSS-NOIDA/issues","login":"DSC-JSS-NOIDA","members_url":"https://api.github.com/orgs/DSC-JSS-NOIDA/members{/member}","node_id":"MDEyOk9yZ2FuaXphdGlvbjEyMzcxNTc4","public_members_url":"https://api.github.com/orgs/DSC-JSS-NOIDA/public_members{/member}","repos_url":"https://api.github.com/orgs/DSC-JSS-NOIDA/repos","url":"https://api.github.com/orgs/DSC-JSS-NOIDA"},{"avatar_url":"https://avatars0.githubusercontent.com/u/12704830?v=4","description":"","events_url":"https://api.github.com/orgs/GT-Vision-Lab/events","hooks_url":"https://api.github.com/orgs/GT-Vision-Lab/hooks","id":12704830,"issues_url":"https://api.github.com/orgs/GT-Vision-Lab/issues","login":"GT-Vision-Lab","members_url":"https://api.github.com/orgs/GT-Vision-Lab/members{/member}","node_id":"MDEyOk9yZ2FuaXphdGlvbjEyNzA0ODMw","public_members_url":"https://api.github.com/orgs/GT-Vision-Lab/public_members{/member}","repos_url":"https://api.github.com/orgs/GT-Vision-Lab/repos","url":"https://api.github.com/orgs/GT-Vision-Lab"},{"avatar_url":"https://avatars1.githubusercontent.com/u/15375248?v=4","description":"","events_url":"https://api.github.com/orgs/jsshack15/events","hooks_url":"https://api.github.com/orgs/jsshack15/hooks","id":15375248,"issues_url":"https://api.github.com/orgs/jsshack15/issues","login":"jsshack15","members_url":"https://api.github.com/orgs/jsshack15/members{/member}","node_id":"MDEyOk9yZ2FuaXphdGlvbjE1Mzc1MjQ4","public_members_url":"https://api.github.com/orgs/jsshack15/public_members{/member}","repos_url":"https://api.github.com/orgs/jsshack15/repos","url":"https://api.github.com/orgs/jsshack15"},{"avatar_url":"https://avatars0.githubusercontent.com/u/20895851?v=4","description":"Building platforms for reproducible AI research","events_url":"https://api.github.com/orgs/Cloud-CV/events","hooks_url":"https://api.github.com/orgs/Cloud-CV/hooks","id":20895851,"issues_url":"https://api.github.com/orgs/Cloud-CV/issues","login":"Cloud-CV","members_url":"https://api.github.com/orgs/Cloud-CV/members{/member}","node_id":"MDEyOk9yZ2FuaXphdGlvbjIwODk1ODUx","public_members_url":"https://api.github.com/orgs/Cloud-CV/public_members{/member}","repos_url":"https://api.github.com/orgs/Cloud-CV/repos","url":"https://api.github.com/orgs/Cloud-CV"}],"organizations_url":"https://api.github.com/users/deshraj/orgs","public_gists":7,"public_repos":110,"received_events_url":"https://api.github.com/users/deshraj/received_events","repos_url":"https://api.github.com/users/deshraj/repos","site_admin":false,"starred_url":"https://api.github.com/users/deshraj/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/deshraj/subscriptions","type":"User","updated_at":"2019-04-12T14:28:52Z","url":"https://api.github.com/users/deshraj"}},"origin":"https://github.com/Cloud-CV/EvalAI","perceval_version":"0.12.10","tag":"https://github.com/Cloud-CV/EvalAI","timestamp":1555132034.144707,"updated_on":1477019450.0,"uuid":"e490cdcd288c5e3c2ed4fe4100042b4cc33e1932"}
```

Comment:
why this file is needed?

Reply:
Sorry, it was for some other checking, by  mistake.

Reply:
Sorry my mistake @valeriocos, it was for some other checking, removed.